cli/config.py

```diff
@@ -160,11 +161,27 @@ def cluster_config_from_dict(config: dict):
         if config.get('password') is not None:
             output.user_configuration.password = config['password']
 
-    if config.get('custom_scripts') not in [[None], None]:
-        output.custom_scripts = config['custom_scripts']
+    if 'custom_scripts' in config:
```

Comment:
Why no `if config.get('custom_scripts) is not None` like the above?

Reply:
Yeah, felt like it was looking cleaner hut this doesn't cover the null case

Reply:
though why was it checking for [[None], None]?